spec/lib/extensions/ar_visible_attribute_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe ArVisibleAttribute do
+  context "visible_attribute_names" do
+    let(:klass) do
+      Class.new(ApplicationRecord) do
+        # ApplicationRecord already included ArVisibleAttribute
+        self.table_name = "vms"
+      end
+    end
+
+    it "shows regular attributes" do
+      expect(klass.visible_attribute_names).to include("format")
+    end
+
+    it "hides hidden attributes" do
+      klass.hide_attribute :format
+      expect(klass.visible_attribute_names).not_to include("format")
+    end
```

Comment:
Can you also add a test that shows that hidden_attribute_names was added to (and is empty to start?)